hadoop/src/test/java/org/apache/carbondata/hadoop/test/util/StoreCreator.java

```diff
@@ -346,26 +346,26 @@ public static void executeGraph(LoadModel loadModel, String storeLocation, Strin
       path.delete();
     }
 
-    DataProcessTaskStatus schmaModel = new DataProcessTaskStatus(databaseName, tableName);
-    schmaModel.setCsvFilePath(loadModel.getFactFilePath());
+    DataProcessTaskStatus schemaModel = new DataProcessTaskStatus(databaseName, tableName);
```

Comment:
revised the value name by your comment, pls check.
CI pass, Update.
http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/471/